### DISCUSSION TITLE
GR-01 ExpressLRS Support for DJI RS4 PRO

### DISCUSSION THREAD

**Discussion by GeordieLillis** [ExpressLRS/ExpressLRS#2743] [General]
Hey guys, 

Chasing a solve to connect the GR-01 ExpressLRS 2.4GHz Receiver to the New DJI RS4 Pro gimbal. 
Currently works fine with the RS2 and RS3... But not getting any love for the 4 Series. 

Can anyone help? 

https://www.radiomasterrc.com/products/gr-01-expresslrs-2-4ghz-receiver-for-dji-rs-camera-gimbals

**Comment by astraw** [comment]
I don't know the answer to your question, but out of curiosity, can you control a DJI focus motor with this receiver? Would this work from an ELRS TX RC? I can't really find much information about the GR-01. What does it do exactly?

**Comment by psionic001** [comment]
Did you end up getting this solved?
I don’t think anything has fundamentally changed on the RS4 side. I used a normal FrSky SBus receiver the other day on RS4 pro and it worked fine. Perhaps possibly they have simplified the system on the DBus input side so it’s not so picky as to the settings for CRSF/ELRS. I remember I tried to get it all working some time ago with CRSF and it was a lot more difficult to get it to play nice than Futaba and FRSky SBus. 

Also when moving from RS3 pro to RS4 pro there was a small difference for switch position on the gimbal and software settings to get the display to say “Custom”. Then it worked flawlessly.

**Reply by llewenda** [reply]
Hi. I can't make it to switch to "custom" mode, can you please help me out?

**Comment by GeordieLillis** [comment]
All resolved. Works perfectly when you adjust the control switch position on the RS4 pro. (Had us confused there for a second)

**Reply by boraka3** [reply]
Hello my RS4 Pro isn't working either.
 Can you elaborate?

**Reply by boraka3** [reply]
I've done all the bindings successfully, but they don't move

**Reply by Spit29** [reply]
@GeordieLillis Just to clarify, I was about to buy a GR-01 ExpressLRS 2.4GHz Receiver for my RS4 Pro. I just wanted to confirm you were able to get it working just by adjusting the control switch? Thanks for further details.
